config/jobs/kubernetes/sig-testing/apidiff.yaml

```diff
@@ -56,5 +56,46 @@ presubmits:
             cpu: 2
             memory: 12Gi
 
+  - name: pull-kubernetes-apidiff-client-go
+    cluster: eks-prow-build-cluster
+    # A job which automatically runs for changes in client-go
+    run_if_changed: '^staging\/src\/k8s.io\/client-go'
+    optional: true
+    decorate: true
+    annotations:
+      # The apidiff.sh script uses the latest revision of apidiff.
+      # There is no guarantee that this will continue to work for
+      # older branches, so let's not even create per-release
+      # copies of this job.
+      fork-per-release: "false"
+      testgrid-dashboards: sig-testing-misc
+      testgrid-create-test-group: 'true'
+    path_alias: k8s.io/kubernetes
+    spec:
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20241125-b4ea3e27a6-master
+        imagePullPolicy: Always
```

Comment:
uh, this should not be always, did you find this in another job?

Reply:
yeah, copy pasted from the same file

Reply:
Which was from me then. But I, too, must have copied it from somewhere else.

Probably pull-kubernetes-verify:
https://github.com/kubernetes/test-infra/blob/ab1dea0a2d9bb1db98ab80b06fbb7de1fd381585/config/jobs/kubernetes/sig-testing/verify.yaml#L20-L21

There's a ton of other hits when grepping for it:
https://github.com/search?q=repo%3Akubernetes%2Ftest-infra%20%22imagePullPolicy%3A%20Always%22&type=code

Reply:
removed here, it seems we need to follow up on that, opened https://github.com/kubernetes/test-infra/issues/33868 for it